app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -55,14 +55,19 @@ keywords:
   - Settlement Scheme
 ---
 
+
+[International relocation payment applications for teachers and salaried trainees who started their job or course in September 2023 are now open](https://get-a-teacher-relocation-payment.education.gov.uk/). You must apply by 31 October 2023.  
```

Comment:
This link looks very long. Is there any way this could be reduced?

Reply:
Tweaked